GameData/Kerbalism/System/Reliability.cfg

```diff
@@ -4,688 +4,944 @@
 {
-  MODULE
-  {
-    name = Reliability
-    type = ModuleDataTransmitter
-    title = Data Transmitter
-    redundancy = Communication
-    repair = Engineer
-    mtbf = 72576000 // 8y
-    extra_cost = 1.0
-    extra_mass = 0.1
-  }
+	MODULE
+	{
+		name = Reliability
+		type = ModuleDataTransmitter
+		title = Data Transmitter
+		redundancy = Communication
+		repair = Engineer
+		mtbf = 72576000 // 8y
+		extra_cost = 1.0
+		extra_mass = 0.1
+	}
 }
 
 @PART[*]:HAS[@MODULE[ModuleDataTransmitter],@MODULE[ModuleCommand],#CrewCapacity[<1]]:NEEDS[FeatureReliability]:FOR[Kerbalism]
 {
-  MODULE
-  {
-    name = Reliability
-    type = ModuleDataTransmitter
-    title = Data Transmitter
-    redundancy = Communication
-    repair = Engineer
-    mtbf = 72576000 // 8y
-    extra_cost = 0.05
-    extra_mass = 0.01
-  }
+	MODULE
+	{
+		name = Reliability
+		type = ModuleDataTransmitter
+		title = Data Transmitter
+		redundancy = Communication
+		repair = Engineer
+		mtbf = 72576000 // 8y
+		extra_cost = 0.5
+		extra_mass = 0.01
+	}
 }
 
 @PART[*]:HAS[@MODULE[ModuleDataTransmitter],@MODULE[ModuleCommand],~CrewCapacity[]]:NEEDS[FeatureReliability]:FOR[Kerbalism]
 {
-  MODULE
-  {
-    name = Reliability
-    type = ModuleDataTransmitter
-    title = Data Transmitter
-    redundancy = Communication
-    repair = Engineer
-    mtbf = 72576000 // 8y
-    extra_cost = 0.05
-    extra_mass = 0.01
-  }
+	MODULE
+	{
+		name = Reliability
+		type = ModuleDataTransmitter
+		title = Data Transmitter
+		redundancy = Communication
+		repair = Engineer
+		mtbf = 72576000 // 8y
+		extra_cost = 0.5
+		extra_mass = 0.01
+	}
 }
 
 @PART[*]:HAS[@MODULE[ModuleDataTransmitter],@MODULE[ModuleCommand],#CrewCapacity[>0]]:NEEDS[FeatureReliability]:FOR[Kerbalism]
 {
-  MODULE
-  {
-    name = Reliability
-    type = ModuleDataTransmitter
-    title = Data Transmitter
-    redundancy = Communication
-    repair = Engineer
-    mtbf = 72576000 // 8y
-    extra_cost = 0.01
-    extra_mass = 0.001
-  }
+	MODULE
+	{
+		name = Reliability
+		type = ModuleDataTransmitter
+		title = Data Transmitter
+		redundancy = Communication
+		repair = Engineer
+		mtbf = 72576000 // 8y
+		extra_cost = 0.05
+		extra_mass = 0.001
+	}
 }
+
+@PART[*]:HAS[@MODULE[Reliability],!MODULE[KerbalEVA]]:AFTER[Kerbalism]
+{
+	@MODULE[Reliability]:HAS[~type[ProcessController]],*
+	{
+		mtbf_nerf = #$/mass$
+		@mtbf_nerf *= #$extra_mass$
+		@mtbf /= #$mtbf_nerf$
+		@mtbf /= 89 // just a good plain Fibonacci number
+	}
+}
+
+@PART[*]:HAS[@MODULE[Reliability],!MODULE[KerbalEVA]]:AFTER[Kerbalism]
+{
+	@MODULE[Reliability]:HAS[#mtbf[<36288000]],*
+	{
+		@mtbf = 36288000 // 4y lowpass filter
+	}
+}
+
+@PART[*]:HAS[@MODULE[Reliability],!MODULE[KerbalEVA]]:AFTER[Kerbalism]
+{
+	@MODULE[Reliability]:HAS[#mtbf[>145152000]],*
+	{
+		@mtbf = 145152000 // 16y highpass filter
+	}
+}
+
+@PART[*]:HAS[@MODULE[Reliability],!MODULE[KerbalEVA]]:FINAL
```

Comment:
Why `FINAL` ?  If it has to go very last then `AFTER[zzzKerbalism]` may be better

Reply:
Ok. I was not sure if a zzz thing even exists, I have not seen any in a quick search...